conf/schema.xml

```diff
@@ -109,7 +109,7 @@
 
          Consult the DateField javadocs for more information.
       -->
-    <fieldType name="date" class="solr.DateField" sortMissingLast="true" omitNorms="true"/>
+    <fieldType name="date" class="solr.TrieDateField" sortMissingLast="true" omitNorms="true" precisionStep="6" positionIncrementGap="100"/>
```

Comment:
Looking a bit more at this... Thinking this should probably stay as `solr.DateField`... Had to be changed for 4.x, since it was deprecated; however, changing the schema without reindexing like this could cause odd behaviours... Like difficult-to-track `NullPointerException`s and junk.